_docs-sources/library/overview/services.md

```diff
@@ -1,16 +1,36 @@
 # What is a service module?
 
-The Gruntwork Service Catalog consists of a number of customizable, production-grade infrastructure-as-code services that you can use to deploy and manage your infrastructure. This includes Docker orchestration, EC2 orchestration, load balancing, networking, databases, caches, monitoring, alerting, CI/CD, secrets management, VPN, and much more. Services combine multiple modules to configure an end-to-end solution.
+**Service modules are opinionated combinations of ["building block" modules](./modules). They are usually written in Terraform, represent a full use case, and are designed to be used "off the shelf."**
```

Comment:
I see what you're trying to do with shortening the description, but I don't think it's quite getting the point across. What about this?

```suggestion
Service modules are combinations of [modules](./modules) covering a wide variety of use cases, including CI/CD, networking, container orchestration (EKS, ECS), and more. Service modules are usually written in Terraform, represent a full use case, and are designed to be used "off the shelf."
```

Regarding style — I don't think we need to put this in bold, it's the first sentence of the first section on the page. We intentionally removed bold text in many other places on the docs site.